core/python_utils_test.py

```diff
@@ -67,7 +67,7 @@ def test_url_unsplit(self):
             python_utils.url_unsplit(response), 'http://www.google.com')
 
     def test_parse_query_string(self):
-        response = python_utils.parse_query_string(
```

Comment:
Are we expecting to remove the `python_utils.parse_query_string` function definition through this PR?

Reply:
Done @DubeySandeep